src/redux/store.js

```diff
@@ -0,0 +1,34 @@
+import { configureStore } from "@reduxjs/toolkit";
+import {
+  FLUSH,
+  PAUSE,
+  PERSIST,
+  PURGE,
+  REGISTER,
+  REHYDRATE,
+  persistStore,
+} from "redux-persist";
+import thunk from "redux-thunk";
+import { setupListeners } from "@reduxjs/toolkit/dist/query";
+
+import accountReducer from "./reducer/accountReducer";
+import configReducer from "./reducer/configReducer";
+
+
+export const store = configureStore({
+  reducer: {
```

Comment:
import these as from `src/reducers.js` ?

Reply:
Yeah i oversaw that those are in reducers.js but normally its named "store" ( as it is the root for the createAPI from toolkit if we want to use it in the future )